ProcessMaker/SanitizeHelper.php

```diff
@@ -78,35 +77,8 @@ private static function convertTagToRegExp($tag)
      *
      * @return string
      */
-    public static function sanitizeData($data, $task)
+    public static function sanitizeData($data, $except = [])
```

Comment:
You are modifying this public method breaking the Open-closed principle, this could cause problems to other projects like:
![image](https://user-images.githubusercontent.com/8028650/188163975-2574bc4d-c36a-4795-86a6-06f00cc19131.png)